common/src/main/kotlin/com/artillexstudios/axminions/minions/Minion.kt

```diff
@@ -640,6 +640,8 @@ class Minion(
 
         val meta = toolMeta as? Damageable ?: return
 
+        if (tool?.itemMeta?.isUnbreakable == true) return
```

Comment:
Could you use the toolMeta variable here? It is quite expensive to get the itemMeta from the item, as spigot clones the meta every time it is accessed.